Sources/Prometheus/RELEASE.md

```diff
@@ -0,0 +1,50 @@
+# Releasing 
+
+## Release schedule
+
+Releases of this library are performed on a when-needed basis. 
+
+Generally one of the maintainers listed in MAINTAINERS.md will perform a release.
+
+If you would like to request a release since a change or fix has been waiting for a release for some time,
+you may open up an issue on github, requesting a release.
+
+## Versioning 
+
+As usual with Swift packages, we use [Semantic Versioning](https://semver.org/).
+
+## Make a release
+
+Development and releases are made from the `main` branch directly, therefore the branch should remain in releasable state at any time.
+
+Making a release is simple and boils down to making a git tag and pushing it to the upstream repository.
+
+Tags should follow the simple `MAJOR.MINOR.PATCH` format, without any prefix (e.g. do not include the `v` prefix).
+
+Step 1) Make a release tag.
+
+```bash
+tag="MAJOR.MINOR.PATCH"
+git tag -s "${tag}" -m "${tag}"
+git push origin "${tag}"
+```
+
+Step 2) Make the GitHub Release and Release Notes.
+
+Release notes generation is automated and based on the enforced labelling of pull requests.
```

Comment:
Would it be possible to suggest some ways to make this paragraph more concise or step-by-step instructional?

Reply:
I'll revisit today then